spec/unit/analytics/google-analytics-universal-tracker.spec.js

```diff
@@ -4,14 +4,14 @@ describe("GOVUK.GoogleAnalyticsUniversalTracker", function() {
   beforeEach(function() {
     window.ga = function() {};
     spyOn(window, 'ga');
-    universal = new GOVUK.GoogleAnalyticsUniversalTracker('id', 'cookie-domain.com');
+    universal = new GOVUK.GoogleAnalyticsUniversalTracker('id', 'cookie-domain.com', { siteSpeedSampleRate: 100 });
   });
 
   it('can load the libraries needed to run universal Google Analytics', function() {
     delete window.ga;
-    $('[src="//www.google-analytics.com/analytics.js"]').remove();
+    $('[src="https://www.google-analytics.com/analytics.js"]').remove();
```

Comment:
Hi @nickcolley,

Technically this line is redundant. Until `GOVUK.GoogleAnalyticsUniversalTracker.load()` runs there's no Google Analytics script in the DOM.

I presume at some point it was a "clean up" before the test runs?
We can remove it if you like.

Otherwise yes, the new instances will always be `https://` as shown here:
https://github.com/alphagov/govuk_frontend_toolkit/pull/298/files#diff-8b93af583c1a7ca5e9c48a89ebce2b38R30